src/logic/wallets/store/middleware/index.ts

```diff
@@ -59,12 +58,6 @@ const providerMiddleware =
     if (available) {
       // Only track when wallet connects to same chain as chain displayed in UI
       if (currentChainId(state) === network) {
-        const event = {
-          ...WALLET_EVENTS.CONNECT_WALLET,
-          label: name,
-        }
-
-        trackAnalyticsEvent(event)
       }
     } else {
       store.dispatch(enqueueSnackbar(enhanceSnackbarForAction(NOTIFICATIONS.UNLOCK_WALLET_MSG)))
```

Comment:
This can be simplified to `if(!available) {store.dispatch...}` now.

Reply:
I left the code because it will be used for GTM tracking.